UI/window-basic-main.cpp

```diff
@@ -173,6 +173,9 @@ static void AddExtraModulePaths()
 				    data_path_with_module_suffix.c_str());
 	}
 
+	if (portable_mode)
+		return;
+
```

Comment:
`portable_mode` has external visibility, so there's no need to pass it to this function, it can be tested directly.

Reply:
you're right, fixed.